src/vivarium_census_prl_synth_pop/components/observers.py

```diff
@@ -171,7 +171,9 @@ def on_simulation_end(self, _: Event) -> None:
             self.responses.index.names = ["simulant_id"]
             filepath = output_dir / f"{self.name}_{self.seed}.{self.file_extension}"
             if "hdf" == self.file_extension:
-                self.responses.to_hdf(filepath, "data", format="table")
+                self.responses.to_hdf(
+                    filepath, "data", format="table", complib="bzip2", complevel=9
```

Comment:
Did you profile these options to land on this complib and complevel?

Reply:
No. If we have time to do profiling later, we will do that, but we will almost certainly want to use hdf files to preserve dtypes, so I'm proactively changing this.